test/sequential/test-net-connect-local-error.js

```diff
@@ -3,25 +3,28 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
-const server = net.createServer();
-server.listen(0);
-const port = server.address().port;
 const client = net.connect({
-  port: port + 1,
-  localPort: port,
+  port: common.PORT + 1,
+  localPort: common.PORT,
   localAddress: common.localhostIPv4
 });
 
 client.on('error', common.mustCall(function onError(err) {
+  assert.strictEqual(err.syscall, 'connect');
+  assert.strictEqual(err.code, 'ECONNREFUSED');
   assert.strictEqual(
     err.localPort,
-    port,
-    `${err.localPort} !== ${port} in ${err}`
+    common.PORT,
+    `${err.localPort} !== ${common.PORT} in ${err}`
   );
   assert.strictEqual(
     err.localAddress,
     common.localhostIPv4,
     `${err.localAddress} !== ${common.localhostIPv4} in ${err}`
   );
-}));
-server.close();
+  assert.strictEqual(
+    err.message,
+    `connect ECONNREFUSED ${err.address}:${err.port} ` +
+    `- Local (${err.localAddress}:${err.localPort})`
+  );
+}));
```

Comment:
Nit: Future reference we require an LF as the last character of all files.
That's why the linter failed (always good to do `make lint`) before a commit
Don't worry about this, I'll fix this when I land the PR.